os_migrate/roles/conversion_host_content/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+- name: display host os
+  debug: msg="Conversion Host OS is {{ ansible_distribution }}."
+
+- name: install epel release for CentOS
+  yum:
```

Comment:
We will probably need to add a become: True in any install task
Then main.yml should call both centos and rhel tasks

Reply:
The become added to the task within the role doesn't work: https://gitlab.cee.redhat.com/os-migrate/bot/-/jobs/1839641

Adding the become at the task where the role is imported doesn't work either: https://gitlab.cee.redhat.com/os-migrate/bot/-/jobs/1839872

My guess is that I need to figure out what user has sudo on the image being used and add that to the end to end testing cfg